input/WIxSC_0.yml

```diff
@@ -0,0 +1,23 @@
+mapper_class: MapperWIxSC
+mask_name: mask_2MPZWISC
+path_rerun: '/mnt/extraspace/damonge/Datasets/2MPZ_WIxSC/xcell_runs'
+coords: G
+z_edges: [0.10, 0.15]
+bin_name: 'z0p10_0p15'
+nside_nl_threshold: 8129
+lmin_nl_from_data: 8192
+nl_analytic: True
```

Comment:
maybe make nl_analytic an object in __init__ for clarity?

Reply:
Not sure what you mean by "an object in init"